### ISSUE TITLE
[Browser] Chrome on iOS not detected

### ISSUE THREAD

**Issue by cossssmin** [kaimallea/isMobile#110]
Using `isMobile().any` or `isMobile().phone` always returns `false`.

Chrome 77.3865.103
iOS 12.4.1

**Comment by cossssmin** [comment]
By the way, this is the user agent I see when I `console.log(navigator.userAgent)`:

```
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_5) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1.1 Safari/605.1.15
```

**Comment by kaimallea** [comment]
Thanks for reporting, will take a look!

**Comment by kaimallea** [comment]
I'm seeing different results. On my iPhone I get back a user agent string that contains both `iPhone` and `CriOS`. The official source code for Chrome also contains documentation backing this: https://chromium.googlesource.com/chromium/src.git/+/master/docs/ios/user_agent.md

Are you sure you haven't clicked "Request Desktop Site" option in Chrome for iOS, which would change the useragent string to impersonate its desktop browser?

**Comment by cossssmin** [comment]
Hmm, interesting. Yes, I’m certain I didn’t click to request desktop, but just in case this was somehow active, let me try again and report back. Thanks!

**Comment by kaimallea** [comment]
Feel free to reopen
